eclair-core/src/main/scala/fr/acinq/eclair/payment/send/Autoprobe.scala

```diff
@@ -96,7 +96,7 @@ object Autoprobe {
     if (peers.isEmpty) {
       None
     } else {
-      peers.drop(secureRandom.nextInt(peers.size)).headOption
+      peers.drop(randomLong().toInt % peers.size).headOption
```

Comment:
We should define a `randomInt(bound: Int)`.

Reply:
This is painful to do if we want an unbiased random, and to be honest we don't need it (the `Autoprobe` is unused or doesn't care about bias). I'd rather add it only if we ever need it...